src/services/feedback.service.ts

```diff
@@ -0,0 +1,84 @@
+import { getAccessToken } from './auth.service'
+import axios from 'axios'
+import { BACKEND_API_URL } from '../config'
+import { getAllCommentsByFeedbackId } from './comments.service'
+import { getAllVotesByFeedbackId, Vote } from './votes.service'
+
+export type Feedback = {
+  id: number
+  title: string
+  body: string
+  commentsCount: number
+  category: string
+  votesCount: number
+}
+
+type Comment = {
+  body: string
+  createdAt: string
+  feedbackId: number
+  id: number
+  parentId: number
+  updatedAt: string
+  userId: number
+}
+
+type FeedbackWithComments = Feedback & {
+  comments: Comment[]
+}
+
+export async function getAllFeedback(): Promise<FeedbackWithComments[]> {
+  try {
+    const resp = await axios.get<Feedback[]>(
+      `${BACKEND_API_URL}/api/feedback/`,
+      {
+        headers: {
+          authorization: `Bearer ${getAccessToken()}`,
+        },
+      }
+    )
+
+    const feedbackArr = resp.data
+    const feedbackIds: number[] = []
+    feedbackArr.forEach((obj) => {
+      feedbackIds.push(obj.id)
+    })
+    const ids = feedbackIds.join()
+
+    const feedbackCommentsVotes = await Promise.all([
```

Comment:
you can destructure the array right here if you wish. it may look like so:
```js
const [commentsList, votesList] = await Promise.all([
      getAllCommentsByFeedbackId(ids),
      getAllVotesByFeedbackId(ids),
    ])
```